controllers/comments-controller.js

```diff
@@ -34,6 +35,20 @@ const getCommentsByArticleId = (req, res, next) => {
     });
 };
 
+const postCommentOnArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  insertCommentOnArticle(article_id, username, body)
+    .then((commentPosted) => {
+      res.status(201).send({ comment: commentPosted[0] });
```

Comment:
We would be best to do [0] on the models side. At this point we want to just be sending an object and it to be clear in the controllers- anything to do with sorting the data and putting it in the correct format for sending we should do in the models 😄

Reply:
Done